core/commands/remove_product_command.rb

```diff
@@ -72,9 +72,14 @@ def remove_product(name)
     role_file_path_config = "#{@mdbci_config.path}/#{name}-config.json"
     target_path_config = "configs/#{name}-config.json"
     extra_files = [[role_file_path, target_path], [role_file_path_config, target_path_config]]
-    @network_config.add_nodes([name])
-    @machine_configurator.configure(@network_config[name], "#{name}-config.json",
-                                    @ui, extra_files)
+    configure_node(name, extra_files)
+  end
+
+  # Configure node
+  # @param node [String] name of the node
+  def configure_node(node, extra_files = [])
```

Comment:
Same comment here. Also it is hard to understand `configure_node` in the context of `remove_product` command.